File: admin/src/main/java/com/jigumulmi/banner/dto/request/UpdateBannerRequestDto.java
package com.jigumulmi.banner.dto.request;

import io.swagger.v3.oas.annotations.media.Schema;
import io.swagger.v3.oas.annotations.media.Schema.RequiredMode;
import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.NotNull;
import jakarta.validation.constraints.Size;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor
public class UpdateBannerRequestDto {

    @NotBlank
    @Size(max = 50)
    private String title;
    @Schema(requiredMode = RequiredMode.REQUIRED)
    @NotNull
    private Boolean isActive;
}

File: admin/src/main/kotlin/com/jigumulmi/banner/dto/request/UpdateBannerRequestDto.kt
package com.jigumulmi.banner.dto.request

import io.swagger.v3.oas.annotations.media.Schema
import io.swagger.v3.oas.annotations.media.Schema.RequiredMode
import jakarta.validation.constraints.NotBlank
import jakarta.validation.constraints.NotNull
import jakarta.validation.constraints.Size

data class UpdateBannerRequestDto (
    val title: @NotBlank @Size(max = 50) String,

    @Schema(requiredMode = RequiredMode.REQUIRED)
    val isActive: @NotNull Boolean
)
